operations/boot_orchestration/Create_a_Session_Template_to_Boot_Compute_Nodes_with_SBPS.md

```diff
@@ -1,52 +1,84 @@
 
 - [Boot set `rootfs_provider` parameter](#boot-set-rootfs_provider-parameter)
-- [Boot set `rootfs_provider_passthrough` parameters](#boot-set-rootfs_provider_passthrough-parameters)
-    - [`<transport>`](#transport)
-    - [`<schema version>`](#schema-version)
-    - [`<IQN Domain>`](#iqn-domain)
-    - [`<DNS SRV record reference>`](#dns-service-srv-record-reference)
-    - [`<client discovery timeout in seconds>`](#client-discovery-timeout-in-seconds)
-    - [`<ramroot>`](#ramroot)
-    - [Example `rootfs_provider_passthrough`](#example-rootfs_provider_passthrough)
-- [Boot set S3 parameters](#boot-set-s3-parameters)
+- [Boot set `rootfs_provider_passthrough` parameters](#boot-set-rootfs_provider_passthrough-parameter)
+    - [`Setting the parameter`](#setting-the-parameter)
+    - [`Detailed Explanation`](#detailed-explanation-of-each-element-of-the-parameter)
+      - [`<transport>`](#transport)
+      - [`<schema version>`](#schema-version)
+      - [`<IQN Domain>`](#iqn-domain)
+      - [`<DNS SRV record reference>`](#dns-service-srv-record-reference)
+      - [`<client discovery timeout in seconds>`](#client-discovery-timeout-in-seconds)
+      - [`<ramroot>`](#ramroot)
+      - [Example `rootfs_provider_passthrough`](#example-rootfs_provider_passthrough)
 - [Example session template input file](#example-session-template-input-file)
-- [Creating a BOS session using the new template](#creating-a-bos-session-using-the-new-template)
 - [Appendix: `root=` kernel parameter](#appendix-root-kernel-parameter)
 
-The Scalable Boot Projection Service (SBPS) is an optional provider for the `rootfs` on compute nodes.
+The Scalable Boot Projection Service (SBPS) is the **default**  provider for the `rootfs` on compute nodes.
+
+Two parameters need to be set to configure SBPS, the `rootfs_provider` and the `rootfs_provider_passthrough`.
 
 ## Boot set `rootfs_provider` parameter
 
 The following value needs to be set in the boot set of the session template in order to make SBPS the `rootfs` provider:
 
 `"rootfs_provider":` Set to `"sbps"`
 
-## Boot set `rootfs_provider_passthrough` parameters
+## Boot set `rootfs_provider_passthrough` parameter
 
-For SBPS, the `rootfs_provider_passthrough` boot set parameter is customized according to the following format:
+### Setting the parameter
+In a BOS session template, the `rootfs_provider_passthrough` parameter should be set to the following string.
 
 ```text
-rootfs_provider_passthrough=<transport>:<schema version>:<IQN Domain>:<DNS SRV record>:<client discovery timeout in seconds>:<ramroot>
+rootfs_provider_passthrough=sbps:v1:iqn.2023-06.csm.iscsi:_sbps-hsn._tcp.my-system.my-site-domain:300
+```
+
+The two parameters/strings that need to be customized are 'my-system' and 'my-site-domain'.
+There are three options to find the values for these parameters/strings.
+
+#### Discovering my-system and my-site-domain
```

Comment:
I see that, options 2 and 3 suggested by David earlier to us were more of usage from the developer's perspective to fetch those details. However, I see that fetching via "craysys" is the best option (referenced in many places of the "docs-csm" which is available to customer). I feel rest of the two options can be removed.

Reply:
I agree. I removed options 2 and 3 and reworded the paragraph.